threads task three.py

```diff
@@ -0,0 +1,45 @@
+"""
+Usage of queues
```

Comment:
Explain what the exercise that you are trying to implement here. Not just "queues usage".